common/defs/duplicity_overrides.json

```diff
@@ -3,9 +3,6 @@
     "erc20:eth:USDT": false,
     "eth:BNB:56": false,
     "eth:MATIC:137": false,
```

Comment:
@matejcik what are those? 

do we still need `misc:BNB` it was removed from fw or not yet?

Reply:
> @matejcik what are those duplicity overrides?

these allow some internal checks to proceed in case two coins have the same ticker symbol. Some of them could probably be removed now that most defs are external

> do we still need e.g. `misc:BNB`, it was removed from fw or not yet?

not yet, we still need it